exercises/concept/paint-by-number/.docs/hints.md

```diff
@@ -0,0 +1,57 @@
+# Hints
+
+## General
+
+- Read about [bitstrings][bitstring] in the Getting Started guide.
+- Review the documentation for the [bitstring special form][bitstring-form].
+- Review [how to convert decimal numbers to binary numbers]([decimal-to-binary-youtube]).
+- Elixir supports [different ways of writing integers][integer-literal], including binary notation.
+
+## 1. Calculate palette bit size
+
+- This task doesn't use the bitstring data type.
+- Find the smallest positive number `n` such that 2 raised to the power of `n` is greater than or equal than the number of colors we need to represent.
+- Create a recursive function for this.
```

Comment:
I did not understand the instructions in th same way. I thought I was supposed to find a direct way.

Reply:
I assume that means you skipped reading the instructions midway and didn't see the sentence "Now, solve this task with recursion and..."?